requirements.txt

```diff
@@ -1,4 +1,4 @@
-cachetools==5.0.0
+cachetools~=5.0.0
```

Comment:
You should read the docs on how `~=` works that were linked in the ticket. It's not as simple as just swapping the version specifier out. https://peps.python.org/pep-0440/#version-specifiers.

We might want to swap the `>=` out for `~=` as well.

Reply:
Oh right because we're saying anything in that major version.

Reply:
I thought about `>=` as well. Any objections @mattp0 @mckadesorensen ?

Reply:
So would we want to go from `x.y.z` to `~=x.y` or `~=x.0`?

Reply:
Says `This operator MUST NOT be used with a single segment version number such as `~=1` but I think `~=1.0` would be valid.

Reply:
Yea, in a perfect world you'd put the minimum compatible version in there for the API that you need. So if something was added in 1.2 that you're using then you should put 1.2, but if you're not using whatever was added in 1.2 then you can put an earlier version such as 1.0. In practice the lower bound doesn't matter quite as much as the upper bound since typically the latest compatible version gets pulled anyway.

Reply:
In that case I feel like I should just try `x.0` and if anything breaks work up from there.

Reply:
> In that case I feel like I should just try x.0 and if anything breaks work up from there.
Not sure I love that idea. If there were dependencies at `x.y.z` then I changed them to `~=x.y` and used `~=x.0` in other cases where possible. My editor was showing vulnerability errors when attempting to use `pip~=19.2`.